.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -167,28 +181,22 @@ jobs:
         env:
           KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
 
-      - name: Build and push kolla overcloud images
+      - name: Build kolla overcloud images
```

Comment:
Will this task now fail if any images failed to build? Whereas IIRC previously it would push all built images, even if some failed.

Reply:
This is now resolved. The action will fail but any built images will be pushed